### ISSUE TITLE
restructure menu

### ISSUE THREAD

**Issue by luto** [Uberspace/lab#369]
Our left-side menu is getting quite large due to the amount of guides (yay!). Rethink the structure.

* show a couple of high-profile guides (wordpress, nextcloud, ... ?)
* otherwise show tags only

**Comment by UXXMOO** [comment]
How about categories?

**Comment by nichtmax** [comment]
Read tags as categories.

So we recently implemented tags, [here](https://lab.uberspace.de/tags) is a first shot. 

Guides are usually tagged with multiple tags, see [WordPress](https://lab.uberspace.de/guide_wordpress.html) for example. 

So in your opinion we should display the tags instead of the list of guides in the menu? Every tag? 

My 2 cents:I think we should approach this like the manual: I want to do stuff and there is a category for it like "content management system" or "blog" and when I click on it it expands and lists the corresponding guides. I don't think we should display _every_ tag like "lang-php". We could link to that from the manual in the PHP article though.

**Comment by UXXMOO** [comment]
Agree. We should handle some tags as categories. Not all. If this is not enough for all the used tags, we need a category above.
